File: sources/server/server/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace server
{
    public class Booking
    {
        public Customer Customer { get; set; }
        public Hotel Hotel { get; set; }
        public String At { get; set; }
        public int Duration { get; set; }
        public int RoomNr { get; set; }
        public int NumAdults { get; set; }
        public int NumChilds { get; set; }

        public Booking() { }

        public Booking(Customer Customer, Hotel Hotel, String At, int Duration, int RoomNr, int NumAdults, int NumChilds)
        {
            this.Customer = Customer;
            this.Hotel = Hotel;
            this.At = At;
            this.Duration = Duration;
            this.RoomNr = RoomNr;
            this.NumAdults = NumAdults;
            this.NumChilds = NumChilds;
        }

    }
}

File: sources/customer/src/com/philippspiess/hotel/api/model/Booking.java
package com.philippspiess.hotel.api.model;

public class Booking {
    public Customer Customer;
    public Hotel Hotel;
    public String At;
    public int Duration;
    public int RoomNr;
    public int NumAdults;
    public int NumChilds;
	
    
    public Customer getCustomer() {
		return Customer;
	}
	public void setCustomer(Customer customer) {
		Customer = customer;
	}
	public Hotel getHotel() {
		return Hotel;
	}
	public void setHotel(Hotel hotel) {
		Hotel = hotel;
	}
	public String getAt() {
		return At;
	}
	public void setAt(String at) {
		At = at;
	}
	public int getDuration() {
		return Duration;
	}
	public void setDuration(int duration) {
		Duration = duration;
	}
	public int getRoomNr() {
		return RoomNr;
	}
	public void setRoomNr(int roomNr) {
		RoomNr = roomNr;
	}
	public int getNumAdults() {
		return NumAdults;
	}
	public void setNumAdults(int numAdults) {
		NumAdults = numAdults;
	}
	public int getNumChilds() {
		return NumChilds;
	}
	public void setNumChilds(int numChilds) {
		NumChilds = numChilds;
	}
	
	public String toString() {
		return Hotel.Name + " at " + At;
	}
}
